library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -196,4 +197,12 @@ else if (s.length() >= 8 && s.length() < 12) {
         finalizedString.deleteCharAt(finalizedString.length()-1);
         return finalizedString.toString();
     }
+    public GradientDrawable SetUpRoundBars(int Color)
```

Comment:
`Color` is used for class name. Change it to `color`
```suggestion
    public GradientDrawable SetUpRoundBars(int color)
```